modelscan/scanners/keras/scan.py

```diff
@@ -29,7 +30,17 @@ def scan(
 
         dep_error = self.handle_binary_dependencies()
         if dep_error:
-            return ScanResults([], [dep_error])
+            return ScanResults(
+                [],
+                [
+                    ModelScanError(
+                        self.name(),
+                        ErrorCategories.DEPENDENCY,
+                        f"To use {self.full_name()}, please install modelscan with dependencies.",
```

Comment:
Should this include the name of the dependency to install?

Reply:
Yes, thank you! Though can't seem to get the pip command nicely in the error description: (tried `{{}} `and escaping with \ but does not seem to work)

```{
    "summary": {
        "total_issues_by_severity": {
            "LOW": 0,
            "MEDIUM": 0,
            "HIGH": 0,
            "CRITICAL": 0
        },
        "total_issues": 0,
        "input_path": "notebooks/KerasModels/unsafe_model.h5",
        "absolute_path": "/Users/mehrinkiani/Documents/modelscan/notebooks/KerasModels",
        "modelscan_version": "0.0.0",
        "timestamp": "2024-02-23T12:17:35.614094",
        "scanned": {
            "total_scanned": 0
        },
        "skipped": {
            "total_skipped": 0,
            "skipped_files": []
        }
    },
    "errors": [
        {
            "category": "DEPENDENCY",
            "description": "To use modelscan.scanners.H5LambdaDetectScan, please install modelscan with h5py extras. 'pip install \"modelscan[[ h5py]]\"' if you are using pip."
        }
    ]
}
```

Reply:
I think because you are using both `'` and `"` in the value of the JSON entry, one of them has to be escaped. Probably the best option is to use backticks instead around pip install:

Also, when I run `pip install modelscan[[h5py]]` or `pip install modelscan[h5py]` neither work, have we not done the work to split these dependencies out into extras?

Reply:
Please try with quotes around `modelscan[h5py]` , i.e. `pip install 'modelscan[h5py]'`

Regarding JSON entry for the error description, thank you for suggesting backticks.  It works for JSON report but not for the console message (`[h5py]` does not get printed). 

JSON report:
```
{
...
    "errors": [
        {
            "category": "DEPENDENCY",
            "description": "To use modelscan.scanners.H5LambdaDetectScan, please install modelscan with h5py extras. `pip install 'modelscan[h5py]'` if you are using pip."
        }
    ]
}
```

Console message: 
```
{
...
    "errors": [
        {
            "category": "DEPENDENCY",
            "description": "To use modelscan.scanners.H5LambdaDetectScan, please install modelscan with h5py extras. `pip install 'modelscan'` if you are using pip."
        }
    ]
}
```

Reply:
> Please try with quotes around modelscan[h5py] , i.e. pip install 'modelscan[h5py]'

Duh, this works, thank you!

> It works for JSON report but not for the console message

Darn, I guess the escaped quotes may be necessary then. Can we just try to have the escapes on the outer quotes rather than the inner quotes? For example: `'\"pip install 'modelscan[h5py]'\"` not `'pip install \"modelscan[h5py]\"'`

This way at least the command will be copy-and-pastable